matcher/service/openreview_interface.py

```diff
@@ -4,38 +4,67 @@
 from tqdm import tqdm
 from matcher.encoder import EncoderError
 from matcher.core import MatcherError, MatcherStatus
+from openreview.api import Note
 
 
 class ConfigNoteInterface:
     def __init__(
         self,
         client,
+        client_v2,
```

Comment:
what if we create two different interface classes? then we can avoid the api version checking.

Reply:
After seeing all of the changes I've had to make for both APIs in this class, maybe this is the best option

Reply:
do you still need to have both client in each interface? I guess having one is enough, right?

Reply:
There is still some information that is only in API1 right like groups? That is why I kept it in but if I'm misunderstanding then I can change this

Reply:
you can use API2 to get groups too.